app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+    # Атрибут класу для зберігання всіх створених екземплярів
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        """
+        Ініціалізує обєкт класу Person.
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        :param name: Імя людини (string).
+        :param age: Вік людини (int).
+        """
+        self.name = name
+        self.age = age
+        # Додаємо новий екземпляр у словник класу
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> None:
+    """
+    Створює список екземплярів класу
+    Person на основі вхідного списку словників.
+    :param people: Список словників, де кожен словник описує людину.
+    :return: Список екземплярів класу Person.
```

Comment:
The function `create_person_list` is documented to return a list of `Person` instances, but it actually returns all instances stored in the `Person.people` dictionary. This might not align with the expected behavior if the function is called multiple times, as it will return all instances created so far, not just those from the current call.